vms/job/tests.py

```diff
@@ -1,15 +1,14 @@
-from django.contrib.auth.models import User
 from django.test import TestCase
+
 from event.models import Event
 from job.models import Job
 from job.services import *
```

Comment:
@vubo I understand that ultimately all the functions from job.services may be tested. Although technically `import *` is ok, it's better practice to actually keep the imports in sync with the actual existing code. While this may seem like a trivial exercise now, it becomes very important if you wish to refactor (or upgrade) in the future. It helps the next developer by not having to second guess what is being used and what is not.